drivers/interrupt_controller/intc_miwu.c

```diff
@@ -186,60 +186,55 @@ int npcx_miwu_interrupt_configure(const struct npcx_wui *wui,
 	const uint32_t base = DRV_CONFIG(miwu_devs[wui->table])->base;
 	uint8_t pmask = BIT(wui->bit);
 
-	if (mode == NPCX_MIWU_MODE_DISABLED) {
-		/* Clear MIWU enable bit */
-		NPCX_WKEN(base, wui->group) &= ~pmask;
```

Comment:
I'm not familiar with Nuvoton Platform and @MulinChao you are officially listed as a collaborator so certainly know it better than me, however `intc_miwu` is used by multiple drivers, e.g. `uart_npcx`, `wdt_npcx` so removing support for `NPCX_MIWU_MODE_DISABLED` only because we modify `gpio_npcx` feels wrong. 

`npcx_miwu_interrupt_configure` API still supports `NPCX_MIWU_MODE_DISABLED` as `mode` argument.

As @pabigot suggested such changes, if justified, should be put in a separate commit.

Reply:
Thanks for the comments. I will remove NPCX_MIWU_MODE_DISABLED and split it into two CLs.

Reply:
Done.